matminer/featurizers/base.py

```diff
@@ -364,10 +364,28 @@ def featurize_many(self, entries, ignore_errors=False, return_errors=False,
                                            return_errors=return_errors,
                                            pbar=pbar)
             with Pool(self.n_jobs) as p:
-                func = partial(self.featurize_wrapper,
+                func = partial(self.featurize_imap_wrapper,
                                return_errors=return_errors,
                                ignore_errors=ignore_errors)
-                return p.map(func, entries, chunksize=self.chunksize)
+                entries = list(
```

Comment:
You can put the comment on the line above and put the rest of this on 1 line instead of 2.